### ISSUE TITLE
TS Error: Cannot find module './lib/openGraphScraper'

### ISSUE THREAD

**Issue by standup75** [jshemas/openGraphScraper#169]
Getting this ts error:

```
Cannot find module './lib/openGraphScraper' or its corresponding type declarations.

3 import setOptionsAndReturnOpenGraphResults from './lib/openGraphScraper';
```

There is a workaround here:

https://github.com/jshemas/openGraphScraper/issues/167

But this is bug that needs attention.
Thank you

**Comment by jshemas** [comment]
Hello,

I have a fix for this in https://github.com/jshemas/openGraphScraper/pull/168 and will publish the fix in the next few days.

**Comment by jshemas** [comment]
Fixed in `open-graph-scraper@5.1.1`

**Comment by standup75** [comment]
That works, thank you
